frontend-react/src/pages/submissions/Submissions.tsx

```diff
@@ -0,0 +1,53 @@
+import { Suspense } from "react";
+import { Helmet } from "react-helmet";
+import { NetworkErrorBoundary } from "rest-hooks";
+
+import { useOrgName } from "../../utils/OrganizationUtils";
+import Spinner from "../../components/Spinner";
+import { ErrorPage } from "../error/ErrorPage";
+
+import SubmissionTable from "./SubmissionsTable";
+
+const OrgName = () => {
```

Comment:
This could be extracted into a component and put in the `src/components`. 
Including the block from the `return` JSX that includes the `Suspense`!
Extra ✨ B O N U S  P O I N T S ✨ if you can use the `Title` component (from our project, _not the USWDS one_) to remedy it. I've used it on the TOS form and success page, but haven't tried plugging the organization name in as the `preTitle`. 

It might work like this
```
const orgName = useOrgName();
[...]
<Title title="Covid-19" preTitle={orgName} />
```
Worth a shot! :)

Reply:
Yeah totally!